src/app.jsx

```diff
@@ -12,6 +13,55 @@ import routes from './routes';
 import analytics from './services/analytics';
 
 const App = () => {
+  // Code for setting up color schemes
+
+  const { mode, setMode } = useColorScheme();
+  // mode listener code from
+  //https://dev.to/gustavogomez/manage-dark-mode-in-reactjs-with-listener-to-detect-dark-mode-in-the-browser-3120
+
+  let colorMode = localStorage.getItem('colorMode') ?? 'system';
+
+  const onSelectMode = (prefersDark) => {
+    prefersDark === 'dark' ? setMode('dark') : setMode('light');
+  };
+
+  const setupColorSchemeListeners = (colorScheme) => {
```

Comment:
You take `colorScheme` as a param here, but within the function you never use it, using the component-scoped variable `colorMode` (which is never updated) instead.

Reply:
Outdated by the custom hook